testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -227,6 +329,96 @@ def test_initial_push_replication(setup_client_syncgateway_test, continuous):
 @pytest.mark.listener
 @pytest.mark.syncgateway
 @pytest.mark.replication
+@pytest.mark.session
+@pytest.mark.parametrize("num_docs, need_attachments, replication_after_backgroundApp", [
+    (1000, True, False),
+    (10000, False, False),
+    (10000, False, False)
+])
+def test_push_replication_with_backgroundApp(setup_client_syncgateway_test, num_docs, need_attachments, replication_after_backgroundApp):
+    """
+    1. Prepare LiteServ to have 10000 documents.
```

Comment:
Use @summary annotation to provide details about module in doc string.

Reply:
done